### PR TITLE
New UI proposition, add links to header

### PR DESCRIPTION
**Pull request by Flightan**
screen shot 2017-11-30 at 15 14 12
screen shot 2017-11-30 at 15 14 40

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -10,9 +10,10 @@ machine:
 
 dependencies:
   post:
-    - npm install saucelabs # allow sending Watai results to SauceLabs
-    - wget https://saucelabs.com/downloads/sc-latest-linux.tar.gz
-    - tar -xzf sc-latest-linux.tar.gz
+    - npm install saucelabs@0.1.1 # allow sending Watai results to SauceLabs
+    # https://wiki.saucelabs.com/display/DOCS/Sauce+Connect+Proxy
+    - curl --remote-name https://saucelabs.com/downloads/sc-4.4.9-linux.tar.gz
+    - tar -xzf sc-4.4.9-linux.tar.gz
```
Comment (Anna-Livia) [inline circle.yml]: 🙌

File: layouts/partials/button-language.html [hunk 2]
```diff
@@ -1,5 +1,11 @@
-{{ if .IsTranslated }}
-    {{ range .Translations }}
-        <a href="{{ .Permalink }}" class="lang">{{ .Site.Language.Lang }}</a>
-    {{ end}}
-{{ end }}
+<ul>
+  <li><a href="{{ "countries" | relLangURL }}">{{ $.Site.Params.countries }}</a></li>
+  <li><a href="{{ "tracking" | relLangURL }}">{{ $.Site.Params.analytics }}</a></li>
```
Comment (Anna-Livia) [inline layouts/partials/button-language.html]: Why put traking in the header ?

**Comment by Anna-Livia** [comment]
- [ ] `Français` instead of `/fr`
- [ ] `Analytics` in footer
- [ ] ssl connection
- [ ] dark blue tittles
- [ ] add weigt to light on dark writing

File: layouts/partials/button-language.html (hunk 2 above)
Comment (Flightan) [reply layouts/partials/button-language.html]: Because I thought it would be cool to have 2 items in the nav bar, but it doesn't make any sense...

**Comment by Flightan** [comment]
@Anna-Livia done !

File: static/css/main.css [hunk 3]
```diff
@@ -1,31 +1,13 @@
...
-header .intro {
-	display: flex;
-	flex-direction: row;
-	align-items: center;
-}
-
 h1 {
 	text-align: center;
 	margin-bottom: 1em;
+	color: #1e3242;
```
Comment (Anna-Livia) [inline static/css/main.css]: This seems to make the baseline "Turn Law into software" disapear :/

Comment (Flightan) [reply static/css/main.css]: 😅

**Comment by Anna-Livia** [comment]
On y est presque :)
capture d ecran 2017-12-12 a 16 56 36
capture d ecran 2017-12-12 a 16 44 48

**Comment by Anna-Livia** [comment]
Que penses-tu de mettre `Turn law into software en plus gras` ?

**Comment by Anna-Livia** [comment]
🙌

File: static/css/main.css [hunk 4]
```diff
@@ -1,31 +1,13 @@
 body {
 	margin: 0;
 	font-size: 140%;
-	background-color: #333;
```
Comment (sandcha) [inline static/css/main.css]: On `http://openfisca.org/en/tracking/`, the tracking status background/iframe is also white. Update it to a light grey or any color that looks good to you?

File: layouts/index.html [hunk 5]
```diff
@@ -7,21 +7,23 @@
...
-			<h1>{{ .Title}}</h1>
+				<h1>{{ .Title}}</h1>
 
-			<a href="{{ "countries" | relLangURL }}" class="cta get-started">{{ $.Param "get_started" }}  🙌</a>
+				<a href="{{ "countries" | relLangURL }}" class="cta get-started">{{ $.Param "get_started" }}  🙌</a>
+			</div>
 		</header>
 
-		<section class="row">
+		<section class="row" id="products">
```
Comment (sandcha) [inline layouts/index.html]: > Add a title for the products section like other pages? A link leading to a section without introduction might seem disturbing.

File: layouts/_default/baseof.html [hunk 6]
```diff
@@ -18,9 +19,9 @@
             <a href="{{ "index.html" | relLangURL }}">
                 <img id="logo-small" src="/img/logo-openfisca.svg" alt="OpenFisca">
             </a>
-            {{ $.Site.Params.pitch }}
+            <h1>{{ $.Site.Params.pitch }}</h1>
         </div>
-       {{ partial "button-language.html" . }}
+       {{ partial "menu-links.html" . }}
```
Comment (sandcha) [inline layouts/_default/baseof.html]: 👍

File: layouts/index.html (hunk 5 above)
Comment (Flightan) [reply layouts/index.html]: Good idea, how should we name it ?
`Les produits qui utilisent OpenFisca` ?
`Powered by Openfisca` ?

File: static/css/main.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (Flightan) [reply static/css/main.css]: Oh I did not notice that the first and last color was the same as the background, neat.

File: layouts/partials/menu-links.html [hunk 7]
```diff
@@ -0,0 +1,11 @@
+<ul>
+  <li><a href="{{ "#products" | relLangURL }}">{{ $.Site.Params.products }}</a></li>
+  <li><a href="{{ "countries" | relLangURL }}">{{ $.Site.Params.countries }}</a></li>
+  <li>
+    {{ if .IsTranslated }}
+        {{ range .Translations }}
+            <a href="{{ .Permalink }}" class="lang">{{ .Site.Language.LanguageName }}</a>
```
[the remark this answers is not in the document]
Comment (Flightan) [reply layouts/partials/menu-links.html]: Well, looks like there was mobile optimisation enabled at all... 
I added the viewport params and fixed some stuff.

Details: https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag

File: layouts/index.html (hunk 5 above)
[the remark this answers is not in the document]
Comment (sandcha) [reply layouts/index.html]: The openfisca red works well!

Comment (sandcha) [reply layouts/index.html]: `Produits de la communauté OpenFisca` / `OpenFisca community products`? @Anna-Livia?
`Powered by OpenFisca` is great but I don't know how to translate it (`Propulsé par OpenFisca` 😅)?

File: layouts/partials/menu-links.html [hunk 8]
```diff
@@ -0,0 +1,11 @@
+<ul>
+  <li><a href="{{ "#products" | relLangURL }}">{{ $.Site.Params.products }}</a></li>
+  <li><a href="{{ "countries" | relLangURL }}">{{ $.Site.Params.countries }}</a></li>
+  <li>
```
[the remark this answers is not in the document]
Comment (sandcha) [reply layouts/partials/menu-links.html]: We would have a menu`Products / Available countries / Documentation`.

File: layouts/_default/baseof.html [hunk 9]
```diff
@@ -7,20 +7,23 @@
 
         <title>OpenFisca - {{ .Title }}</title>
 
+        <link href="/css/header.css" rel="stylesheet" type="text/css">
         <link href="/css/main.css" rel='stylesheet' type="text/css" />
         <link href="/css/countries.css" rel='stylesheet' type="text/css" />
         <link href="/css/tracking.css" rel='stylesheet' type="text/css" />
         <link href="/css/legal-notice.css" rel='stylesheet' type="text/css" />
+
+    		<meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (sandcha) [inline layouts/_default/baseof.html]: Typo/indentation.

File: layouts/index.html (hunk 5 above)
Comment (sandcha) [reply layouts/index.html]: Team validated:
FR: `Propulsé par OpenFisca` 🚀 
EN: `Powered by OpenFisca` 🚀

File: layouts/partials/footer.html [hunk 10]
```diff
@@ -1,7 +1,6 @@
 <footer>
     <ul>
         <li><a href="{{ "index.html" | relLangURL }}">{{ $.Site.Params.index }}</a></li>
-        <li><a href="{{ "countries" | relLangURL }}">{{ $.Site.Params.countries }}</a></li>
```
[the remark this answers is not in the document]
Comment (sandcha) [reply layouts/partials/footer.html]: @Flightan Team validated:
`Contact us / Contribute to this page / Analytics policy / Legal notice`

File: layouts/partials/menu-links.html (hunk 8 above)
[the remark this answers is not in the document]
Comment (Flightan) [reply layouts/partials/menu-links.html]: Done !

File: layouts/partials/footer.html (hunk 10 above)
[the remark this answers is not in the document]
Comment (Flightan) [reply layouts/partials/footer.html]: Done !

File: layouts/index.html (hunk 5 above)
Comment (Flightan) [reply layouts/index.html]: Done !

**Pull request merged**